### PR TITLE
Add products in list

### PR DESCRIPTION
**Pull request by Ponram2k**
- [x] Add the product to the list feature is added
- [x] Validation is added for the product details should not empty and null
- [x] Validation is added for the product rate and quantity not lesser than one

### CODE REVIEW

File: src/main/java/in/ponram/servelet/AddProductServelet.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package in.ponram.servelet;
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/ponram/servelet/AddProductServelet.java:1]: Rename to "servlet"

File: src/main/java/in/ponram/servelet/AddProductServelet.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		String itemId = request.getParameter("itemId");
+		String brandName = request.getParameter("brandName");
+		String itemName = request.getParameter("itemName");
+		String itemCategory = request.getParameter("itemCategory");
+		int rate = Integer.parseInt(request.getParameter("rate"));
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/ponram/servelet/AddProductServelet.java:29]: Possibility of parsing exception. Please add try catch block.

File: src/main/java/in/ponram/service/ProductManager.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+import in.ponram.model.Product;
+import in.ponram.validator.ProductValidation;
+
+public class ProductManager {
+
+	private static List<Product> stockDetails = new ArrayList<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/ponram/service/ProductManager.java:11]: Add final

File: src/test/java/in/ponram/util/TestNumberValidator.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	 */
+	// if given number is less than 0
+	@Test
+	public void testNumberIsLessThanZero() {
+
+		boolean valid = NumberValidator.isValidNumber(-1);
+		assertFalse(valid);
+	}
+
+	// if given number is equals to 0
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/ponram/util/TestNumberValidator.java:20]: Use Method comments(/** ) instead of single line comment

**Review by ghost — CHANGES_REQUESTED** [review]
Comments Added

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**